pkg/policy/api/cidr.go

```diff
@@ -196,6 +196,8 @@ func (s CIDRRuleSlice) GetAsEndpointSelectors() EndpointSelectorSlice {
 				Key:      lbl.GetExtendedKey(),
 				Operator: slim_metav1.LabelSelectorOpExists,
 			})
+		} else if r.CIDRGroupSelector != nil {
```

Comment:
nit: There is an existing unit test for this function. It would be great to have this case tested as well.